src/styles/utils/variables.scss

```diff
@@ -0,0 +1,4 @@
+$block-size: 100px;
+$block-gap: 1px;
+$grey-color: #eee;
```

Comment:
We are using this color only once. So i think there is no need to make separete variable for it.